store/store_asserts.go

```diff
@@ -132,8 +123,8 @@ func (s *Store) downloadAssertions(u *url.URL, decodeBody func(io.Reader) error,
 				// default error handling
 
 				// is it v2 error?
```

Comment:
maybe make turn this code into:  assertsionSvcError.toError() returning an error